docs/wcs/examples/programmatic.py

```diff
@@ -18,7 +18,10 @@
 w.wcs.set_pv([(2, 1, 45.0)])
 
 # Some pixel coordinates of interest.
-pixcrd = np.array([[0, 0], [24, 38], [45, 98]], np.float_)
+# Note also that the pixel coordinates are pairs of [X, Y]
+# FITS WCS usually uses [1, 1] as the origin, as it hails from the Fortran tradition.
```

Comment:
Just want to be consistent with above...
```suggestion
# and since WCS built from FITS header automatically has
# origin set to 1, [0, 0] is actually not inside the image.
```